### ISSUE TITLE
Broken on OSX

### ISSUE THREAD

**Issue by acertain** [doy/rust-term#1]
```
rustc --out-dir lib -L tmp src/term.rs && touch tmp/built
error: linking with `cc` failed with code 1
note: cc arguments: -L/usr/local/lib/rustc/x86_64-apple-darwin/lib -m64 -o lib/libterm-e8871eeb2d6d7be5-0.0.1.dylib lib/term.o -L/usr/local/lib/rustc/x86_64-apple-darwin/lib -lcore-c3ca5d77d81b46c1-0.6 -Ltmp -lio_helper -lcurses -lcurses_helper -ltermios_wrapper -dynamiclib -Wl,-install_name,@rpath/libterm-e8871eeb2d6d7be5-0.0.1.dylib -Wl,-no_compact_unwind -lmorestack -lrustrt -Wl,-rpath,@executable_path/../../../../../../../usr/local/lib/rustc/x86_64-apple-darwin/lib -Wl,-rpath,/usr/local/lib/rustc/x86_64-apple-darwin/lib -Wl,-rpath,/usr/local/Cellar/rust/0.6/lib/rustc/x86_64-apple-darwin/lib
note: Undefined symbols for architecture x86_64:
  "_tiparm", referenced from:
      _tiparm1 in libcurses_helper.a(curses_helper.o)
      _tiparm2 in libcurses_helper.a(curses_helper.o)
     (maybe you meant: _tiparm2, _tiparm1 )
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

error: aborting due to previous error
make: *** [tmp/built] Error 101
```

**Comment by doy** [comment]
Do you have ncurses installed? Which package/version?

**Comment by acertain** [comment]
I have ncurses-5.4 and also tried it with ncurses-5.9 from homebrew.

**Comment by doy** [comment]
Fixed by 7628b3c.
